packages/plugins/src/cli/deploy.ts

```diff
@@ -185,17 +176,8 @@ export const deploy = async (
     forgeScriptCollectArgs.push('--target-contract', targetContract)
   }
 
-  // Collect the transactions. We use the `FOUNDRY_SENDER` environment variable to set the
-  // Gnosis Safe as the `msg.sender` to ensure that it's the caller for all transactions. We need
-  // to do this even though we also broadcast from the Safe's address in the script.
-  // Specifically, this is necessary if the user is deploying a contract via CREATE2 that uses a
-  // linked library. In this scenario, the caller that deploys the library would be Foundry's
-  // default sender if we don't set this environment variable. Note that `FOUNDRY_SENDER` has
-  // priority over the `--sender` flag and the `DAPP_SENDER` environment variable.
-  const dateBeforeForgeScriptCollect = new Date()
-  const spawnOutput = await spawnAsync('forge', forgeScriptCollectArgs, {
-    FOUNDRY_SENDER: safeAddress,
-  })
```

Comment:
Had to remove the `FOUNDRY_SENDER` here and in the Propose command to avoid an edge case that occurred because I merged the collection and gas estimation steps. It's fine to remove the `FOUNDRY_SENDER` because we don't support linked libraries anyways.

If you're curious, the edge case is that Foundry throws an error if we attempt to deploy a contract at the same address as the `FOUNDRY_SENDER`. In the context of this Forge script call, we were setting the `FOUNDRY_SENDER` to be the Gnosis Safe, and we're also deploying the Gnosis Safe in `Sphinx.sol:sphinxCollect`.

had to remove the `FOUNDRY_SENDER` b/c of a bug where the `createProxyWithNonce`
  // would fail b/c a deployed address can't be the sender. was happening on `createProxyWithNonce`,
  // where we deploy the safe.